interp/dumpglob.ml

```diff
@@ -41,6 +41,17 @@ let pop_output () = glob_output := match !glob_output with
 let pause () = push_output NoGlob
 let continue = pop_output
 
+let without_glob f () =
+  push_output NoGlob;
```

Comment:
```suggestion
let with_output g f () =
  push_output g;
```
?

Reply:
Yes, good idea. I'd just propose to name it `with_glob_output` since there is already a `Topfmt.with_output_to`??

Reply:
Do as you prefer.